### DISCUSSION TITLE
Missing input_binary

### DISCUSSION THREAD

**Discussion by openPhiL** [zachowj/node-red-contrib-home-assistant-websocket#772] [Feature Request]
Hi. I can see lots of input actions available here: 
![image](https://user-images.githubusercontent.com/43743230/203495703-a55e80d1-4bdc-4e8d-97a0-58ab499e34c5.png)

but to change the state of my motion sensors (for simulation purpose, running unit-tests), it seems that I need the option **input_binary**, which is not available. 
It would do something like this:
![image](https://user-images.githubusercontent.com/43743230/203495900-c2699d54-da05-4d93-9a0d-733989e7d3d9.png)
![image](https://user-images.githubusercontent.com/43743230/203495924-99992966-7775-4233-9eaa-2c72f5e909c7.png)

would be great if that is just a simple thing you could add. 
Thanks
PhiL

**Comment by zachowj** [comment]
Home Assistant doesn't have an `input_binary` domain. If you want to update a binary sensor for testing you would need to use the [REST API](https://developers.home-assistant.io/docs/api/rest)
